app/views/projects/samples/metadata/_edit_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_with(model: @sample, url: namespace_project_sample_metadata_field_path(sample_id: @sample.id), method: :patch) do |form| %>
+  <%= turbo_frame_tag 'update-alert' %>
+  <div class="grid gap-4">
+  <%= form.fields_for :edit_field do |field| %>
+    <%= field.fields_for :key do |metadata_key| %>
+      <div class="form-field">
+        <%= metadata_key.label t("projects.samples.show.metadata.edit.key"),
+                  class:
+                    "block mb-2 text-sm font-medium text-slate-900 dark:text-white" %>
+        <%= metadata_key.text_field key,
+                        value: key,
+                        required: true,
+                        class:
+                          "block w-full mb-2 text-xs text-slate-900 border border-slate-300 rounded-lg cursor-pointer bg-slate-50 dark:text-slate-400 focus:outline-none dark:bg-slate-700 dark:border-slate-600 dark:placeholder-slate-400" %>
+      </div>
+    <% end %>
+     <%= field.fields_for :value do |metadata_value| %>
+      <div class="form-field">
+        <%= metadata_value.label t("projects.samples.show.metadata.edit.value"),
+                  class:
+                    "block mb-2 text-sm font-medium text-slate-900 dark:text-white" %>
+        <%= metadata_value.text_field value,
+                        value: value,
+                        required: true,
```

Comment:
**question:** @ericenns how do you feel about a user wanting to remove a value from this, i.e. clearing the input?  Or are we just going to have a delete option?